Pillar_II/esm/src/launch_fesom2_cont.sh

```diff
@@ -152,9 +188,9 @@ echo "MEMBERS         : ${MEMBERS}"
 # shellcheck source=src/fesom2/env/mn4.sh
 source "${SCRIPT_DIR}/${MODEL}/env/${HPC}_container.sh"
 
-CONTAINER=/work/ab0995/eflows4hpc/images/pillar_ii_esm_skylake_openmpi_4_nogpu_v_latest.sif
-DATA_PATH=/work/ab0995/eflows4hpc/data/
-OUT_PATH=/work/ab0995/eflows4hpc/output/
+#CONTAINER=/work/ab0995/eflows4hpc/images/pillar_ii_esm_skylake_openmpi_4_nogpu_v_latest.sif
+#DATA_PATH=/work/ab0995/eflows4hpc/data/
+#OUT_PATH=/work/ab0995/eflows4hpc/output/
```

Comment:
:+1: I think these commented out values may be useful for quick tests, so I think we can leave these ones.